EstateReporting.IntegrationTests/Shared/SharedSteps.cs

```diff
@@ -0,0 +1,749 @@
+            serialisedMessage.Metadata.Add(MetadataContants.KeyNameMerchantId, merchantId.ToString());
+            serialisedMessage.SerialisedData = JsonConvert.SerializeObject(logonTransactionRequest, new JsonSerializerSettings
+                                                                                                    {
+                                                                                                        TypeNameHandling = TypeNameHandling.All
+                                                                                                    });
+
+            SerialisedMessage responseSerialisedMessage =
+                await this.TestingContext.DockerHelper.TransactionProcessorClient.PerformTransaction(this.TestingContext.AccessToken,
+                                                                                                     serialisedMessage,
+                                                                                                     cancellationToken);
+
+            return responseSerialisedMessage;
+        }
+
+        private async Task<SerialisedMessage> PerformSaleTransaction(Guid estateId, Guid merchantId, DateTime transactionDateTime, String transactionType, String transactionNumber, String deviceIdentifier, String operatorIdentifier, Decimal transactionAmount, String customerAccountNumber, String customerEmailAddress,
+                                                                     Guid contractId,
+                                                                     Guid productId,
+                                                                     CancellationToken cancellationToken)
+        {
+            SaleTransactionRequest saleTransactionRequest = new SaleTransactionRequest
+                                                            {
+                                                                MerchantId = merchantId,
+                                                                EstateId = estateId,
+                                                                TransactionDateTime = transactionDateTime,
+                                                                TransactionNumber = transactionNumber,
+                                                                DeviceIdentifier = deviceIdentifier,
+                                                                TransactionType = transactionType,
+                                                                OperatorIdentifier = operatorIdentifier,
+                                                                AdditionalTransactionMetadata = new Dictionary<String, String>
+                                                                                                {
+                                                                                                    {"Amount", transactionAmount.ToString()},
```

Comment:
Codacy found an issue: [Define the locale to be used in this string operation.](https://app.codacy.com/manual/stuart_ferguson1/EstateReporting/pullRequest?prid=6339123)